theme/theme.scss

```diff
@@ -29,3 +29,14 @@
 [class*="ck-heading_heading"] {
 	font-weight: bold;
 }
+
+// Resize dropdown's button label.
+.ck-dropdown {
+	&.ck-heading-dropdown {
+		.ck-dropdown__button {
+			.ck-button__label {
+				width: 8em;
```

Comment:
Besides, I think we need `width`. Without it, the dropdown will change its dimensions as the text of the button is replace, e.g. when the user selects some option.